homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -151,7 +151,7 @@ def unit_of_measurement(self) -> str:
     @property
     def device_state_attributes(self):
         """Return the state attributes of the security zone group."""
-        return {ATTR_MODEL_TYPE: self._device.modelType}
+        return {ATTR_MODEL_TYPE: self._device.modelType, ATTR_IS_GROUP: False}
```

Comment:
According to the docstring this is a group.

Reply:
Good catch. I fixed the docstring.